css-flexbox-photo-gallery/css/style10.css

```diff
@@ -0,0 +1,40 @@
+* {
```

Comment:
This file could start with the code from the last step that learners should have entered, and instructions for this step:

```
* {
  box-sizing: border-box;
}

/*
Remove the default margin from all elements on the page by creating a new rule targeting `body`.
Inside the new rule, set the `margin` to 0.
*/
```